src/extension/src/Constants.py

```diff
@@ -55,14 +58,20 @@ def __iter__(self):
     ENABLE_MAX_RUNTIME = 3
     DISABLE_MAX_RUNTIME = 13
 
-    # Todo: will be implemented later
-    # Telemetry Categories
-    TelemetryExtState = "State"
-    TelemetryConfig = "Config"
-    TelemetryError = "Error"
-    TelemetryWarning = "Warning"
-    TelemetryInfo = "Info"
-    TelemetryDebug = "Debug"
+    # Telemetry Settings
+    TELEMETRY_MSG_SIZE_LIMIT_IN_CHARACTERS = 3072  # 3KB
+    TELEMETRY_EVENT_SIZE_LIMIT_IN_CHARACTERS = 6144  # 6KB
+    TELEMETRY_EVENT_FILE_SIZE_LIMIT_IN_CHARACTERS = 4194304  # 4MB
+    TELEMETRY_DIR_SIZE_LIMIT_IN_CHARACTERS = 41943040  # 40MB
```

Comment:
IN_BYTES -- all lines 62 to 65.  Bytes per character vary based on encoding used from 1 to 4 bytes. So characters would not be the correct measurement here.